src/app/manage/page.tsx

```diff
@@ -82,14 +81,10 @@ export default async function ManagePage({ searchParams }: { searchParams: { tok
         )}
         <Stack direction="row" columnGap={4}>
           {settings && settings.includes(ProfileLinks.PaymentMethod) && (
-            <SimpleButton>
-              <Typography variant="md">Set a payment method</Typography>
-            </SimpleButton>
+            <Button route={'billing'}>Set a payment method</Button>
```

Comment:
IMO `Button` component having prop named `route` is not quite semantic @rrojan

Reply:
I have renamed the prop to better reflect what the purpose of `route` is for now